### PR TITLE
feat: add loan_amount to Account model

### PR DESCRIPTION
**Pull request by tessstoddard**
# Summary of Changes

Add loan_amount to the Account model. This field is already in the spec https://mxcom.atlassian.net/browse/GCU-409

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by stevecl5 — APPROVED** [review]
LGTM! This will need a code owner approval, though.

**Review by mattnichols — APPROVED** [review]

**Pull request merged**
